README.md

```diff
@@ -121,70 +100,164 @@
+
+The next step is to configure your project, and it's different for every e-commerce integration. Go to the [Integrations](https://docs.vuestorefront.io/v2/integrations/) page, open the documentation for the integration you selected in our CLI, and look for the page describing the configuration steps. Or just click on the integration icon below:
+
+<!-- add icons for integrations leading to docs -->
+<p align="center" valign="middle">
+    <a title="Magento" target="_blank" href="https://docs.vuestorefront.io/magento/installation-setup/configure-magento.html">
+        <span valign="center">
+            <img alt="Magento" width="55" src="https://user-images.githubusercontent.com/1626923/156659008-edbd2bed-74d5-4f0d-8a4a-eab56c283d24.png" />
+        </span>
+     </a>
+    &nbsp;&nbsp;
+    <a title="Shopify" target="_blank" href="https://docs.vuestorefront.io/shopify/guide/getting-started.html">
+        <img alt="Shopify" width="55" src="https://user-images.githubusercontent.com/1626923/156658959-d8da60fd-9904-44f4-9823-11a3e821c155.png" />
+    </a>
+    &nbsp;&nbsp;
+    <a title="Odoo" target="_blank" href="https://docs.vuestorefront.io/odoo/">
+    <span valign="center">
+        <img alt="Odoo" width="55" src="https://user-images.githubusercontent.com/1626923/156659136-55c8d696-febd-410e-ab72-b217e9e745e6.png" />
+    </span>
+    </a>
+    &nbsp;&nbsp;
+    <a title="Prestashop" target="_blank" href="https://docs.vuestorefront.io/prestashop/guide/getting-started.html">
+        <img alt="Prestashop" width="55" src="https://user-images.githubusercontent.com/1626923/156659137-d857ad24-f23f-4c80-ac56-4ef6c0721274.png" />
+    </a>
+    &nbsp;&nbsp;
+    <a title="Shopware" target="_blank" href="https://shopware-pwa-docs.vuestorefront.io/">
+        <img alt="Shopware" width="55" src="https://user-images.githubusercontent.com/1626923/156659145-d98ea807-872c-42a9-8fdb-4c4f30a68895.png" />
+    </a>
+    &nbsp;&nbsp;
+    <a title="Spree" target="_blank" href="https://docs.vuestorefront.io/spree/guide/getting-started.html">
+        <img alt="Spree" width="55" src="https://user-images.githubusercontent.com/1626923/156659147-5576eb2f-4cf8-4610-8631-18d03e3f52de.png" />
+    </a>
+    &nbsp;&nbsp;
+    <a title="Sylius" target="_blank" href="https://github.com/vuestorefront-community/sylius">
+        <img alt="Sylius" width="55" src="https://user-images.githubusercontent.com/1626923/161141058-8f6a4a8e-2c6f-402e-b19b-892c7acec77f.png" />
+    </a>
+    &nbsp;&nbsp;
+    <a title="Vendure" target="_blank" href="https://docs.vuestorefront.io/vendure/guide/configuration.html#steps">
+        <img alt="Vendure" width="55" src="https://user-images.githubusercontent.com/1626923/156659153-fafdeb94-a34e-4e9e-8e8a-0f03afde9741.png" />
+    </a>
+<br /><br />
+  <a target="_blank" href="https://docs.vuestorefront.io/v2/integrations/#ecommerce-platforms">Check all available integrations</a>
 </p>
 
 
-## Links
+### **Step 4: Start the project** 🚀
```

Comment:
I would get rid of step 4 and 5 as they are already part of the per-integration guide